php/docker/templates/nginx-vhost-conf.d/75-page-cache-locations.conf

```
@@ -0,0 +1,52 @@
+# vim: set ft=nginx:
+
+{{- if eq .Env.STACK_PAGE_CACHE_BACKEND "redis" }}
+location /.stack-cache-fetch {
+    internal;
+
+    set $redis_key $args;
+    redis_pass {{ default "localhost:6379" .Env.STACK_PAGE_CACHE_REDIS_HOST_PATH }};
+}
+
+location /.stack-cache-store {
+    internal;
+
+    set_unescape_uri $exptime $arg_exptime;
+    set_unescape_uri $key $arg_key;
+
+    redis2_query set $key $echo_request_body;
+    redis2_query expire $key $exptime;
+    redis2_pass {{ default "localhost:6379" .Env.STACK_PAGE_CACHE_REDIS_HOST_PATH }};
+}
+{{- end }}
+
+{{- if eq .Env.STACK_PAGE_CACHE_BACKEND "memcached" }}
```

For clarity I think it's best to use `else if`
```suggestion
{{- else  if eq .Env.STACK_PAGE_CACHE_BACKEND "memcached" }}
```